Make class name requirement clearer (#33)
IntelliTect-Samples/EWU-CSCD371-2024-Fall: README.md

## Assignment

> - There is an existing `BaseLogger` class. It needs an **auto property** to hold a class name. (The class name it holds is determined by which class calls `LogFactory`'s `CreateLogger` method. This class - that calls the `CreateLogger` method - will pass in its name using nameof(), as specified below.) This property should be set in the `LogFactory` using an **object initializer**. ❌✔

Reviewer: ```suggestion
- There is an existing `BaseLogger` class. It needs an **auto property** to store class name -  the name of the class that calls the `LogFactory`'s `CreateLogger()` method. This class (that calls the `CreateLogger()` method) will identify its name using nameof(), as specified below.) This property should be set in the `LogFactory` using an **object initializer**. ❌✔
```

Author: Thanks for the suggestions, Mark. It's more clear now, and I'll commit it!
